samples/sampleLib.py

```diff
@@ -0,0 +1,58 @@
+import json
+import vm_automation
+
+def checkForProcess(vmObject, processName):
+    vmObject.updateProcList()
+    if processName in ' '.join(vmObject.procList):
+        return True
+    else:
+        return False
+
+def getVmObjectFromName(vmServer, vmName):
```

Comment:
This seems like it might be something to move into `esxiServer` class as `esxiServer.findVmByName(vm_name)` or something of that nature.

Reply:
I agree